man/pattern2list.Rd

```diff
@@ -42,17 +43,18 @@ Function to assign multiple slots to a S4 object
 \code{\link[=pattern2id]{pattern2id()}}
 }
 \keyword{Convert}
+\keyword{Used}
+\keyword{[kwic].}
+\keyword{[tokens_compound],}
+\keyword{[tokens_select],}
```

Comment:
This is causing one of the R-CMD-CHECK tests to fail on CI. But it should not be generating these keywords at all. Something is wrong with the Rd here and I think it has to do with the undocumented arguments. I'm not sure why the CRAN tests are not showing that several parameters are undocumented, but I think if you document them, it will fix this, and then the last test will pass.